src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -202,7 +202,7 @@ public static IResourceBuilder<T> WithPgAdmin<T>(this IResourceBuilder<T> builde
                         writer.WriteString("Name", postgresInstance.Name);
                         writer.WriteString("Group", "Servers");
                         writer.WriteString("Host", endpoint.ContainerHost);
-                        writer.WriteNumber("Port", endpoint.Port);
+                        writer.WriteNumber("Port", (decimal)endpoint.ContainerPort!);
```

Comment:
Why is this casting to decial?

Reply:
The linter got me confused; updated to cast to int